rewrite-toml/src/main/antlr/TomlParser.g4

```diff
@@ -124,7 +123,7 @@ table
     ;
 
 standardTable
-    : L_BRACKET key R_BRACKET (commentOrNl* expression)*
+    : L_BRACKET key R_BRACKET (commentOrNl* keyValue)*
```

Comment:
callout: For the TOML language this isn't correct because TOML allows tables to also contain other nested tables and arrays of tables as well. So as a result of this change, we're temporarily artificially limiting the TOML files that we can parse.

Reply:
As discussed I think the nesting here is only "conceptual" and that the language structure and therefore LST model is indeed flat. We could of course provide a JSON mapping or similar if client code needs access to this conceptual model.